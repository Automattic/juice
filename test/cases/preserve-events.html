<html><body>
<a href="#" onclick="return false;">Google</a>
<p onclick="return false;">Google</p>
</body></html>
