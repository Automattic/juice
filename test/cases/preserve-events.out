<html><body>
<a href="#" onclick="return false;">Google</a>
<p onclick="return false;" style="color: red;">Google</p>
</body></html>
